hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMStateMachine.java

```diff
@@ -394,6 +394,10 @@ public void close() throws IOException {
     transactionBuffer.close();
     HadoopExecutors.
         shutdown(installSnapshotExecutor, LOG, 5, TimeUnit.SECONDS);
+    if(!scm.isRunForTest()) {
+      ExitUtils.terminate(1, "ScmStateMachine is closed, shutdown SCM",
+          StateMachine.LOG);
+    }
```

Comment:
I don't think we need this new `isRunForTest` flag.  `ExitUtils.disableSystemExit()` serves exactly the same purpose (see 30cb5e764 for similar change in integration tests).

Reply:
thanks ,  i will take a look